test/functional/interface_bitcoin_cli.py

```diff
@@ -381,6 +381,10 @@ def run_test(self):
         assert_raises_process_error(1, "Could not connect to the server", self.nodes[0].cli('-rpcwait', '-rpcwaittimeout=5').echo)
         assert_greater_than_or_equal(time.time(), start_time + 5)
 
+        self.log.info("Test that only one of -addrinfo, -generate, -getinfo, -netinfo may be specified at a time")
+        assert_raises_process_error(1, "Only one of -getinfo, -netinfo may be specified", self.nodes[0].cli('-getinfo', '-netinfo').send_cli)
```

Comment:
36b21dd: nit: I think any 1 assertion would suffice since there's no extra coverage from the second assertion?

Reply:
initially added two assertions to cover different edge cases, but I see how that can be redundant, I've deleted one.